src/ProjectTemplates/Web.ProjectTemplates/Api-CSharp.csproj.in

```diff
@@ -0,0 +1,17 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFramework>${DefaultNetCoreTargetFramework}</TargetFramework>
```

Comment:
Is there a way to say that this template only supports `net8.0`+? I'm not familiar with the templating stuff, so if it is already handled, disregard.

At a minimum, I wouldn't expect `net7.0` + `-aot` to be supported.

Reply:
That's already handled in the `template.json` file:

```json
  "Framework": {
      "type": "parameter",
      "description": "The target framework for the project.",
      "datatype": "choice",
      "choices": [
        {
          "choice": "net8.0",
          "description": "Target net8.0"
        }
      ],
      "replaces": "net8.0",
      "defaultValue": "net8.0"
    },
```